2018-09-05 10:29:05,557 - DataBase.py[line:23] - INFO: database class is creating...
2018-09-05 10:29:05,640 - DataBase.py[line:51] - INFO: database is connecting...
2018-09-05 10:29:20,459 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_name=name1
2018-09-05 10:29:20,511 - DataBase.py[line:62] - ERROR: query database exception, (1054, "Unknown column 'name1' in 'where clause'")
